app/models/unlocalized_words.php

```diff
@@ -0,0 +1,104 @@
+<?php
+namespace Transvision;
+
+// Build arrays for the search form
+$channel_selector = Utils::getHtmlSelectOptions(
+    $repos_nice_names,
+    $repo,
+    true
+);
+$target_locales_list = Utils::getHtmlSelectOptions(
+    Project::getRepositoryLocales($repo),
+    $locale
+);
+
+$ref_locale = Project::getReferenceLocale($repo);
+$unlocalized_words = [];
+
+/*
+    pspell helps getting rid of false positive results by keeping only valid
+    English words. The downside is that it’s filtering out 'jargon' words that
+    can be used in devtools or Mozilla-specific words.
+*/
+if (! $pspell_link = \pspell_new('en_US', '', '', '', PSPELL_FAST)) {
```

Comment:
I doubt it, it should throw an exception since the class is not available.

This should work:
```PHP
if (! class_exists('\pspell_new')) {
    // Do something
} else {
...
}
```

And I would display something to the user, I think we're giving a white page right now.